.evergreen/compile-unix.sh

```diff
@@ -28,6 +29,7 @@ set -o errexit  # Exit the script with error if any of the commands fail
 #       ZSTD                    Build against system zstd.
 
 # Options for this script.
+C_STD_VERSION=${CSTD_VERSION:-99}
```

Comment:
@eramongodb: I think this was a typo, but it appears to have been fixed in a subsequent commit: https://github.com/mongodb/mongo-c-driver/commit/f4a83c084088e3878097b17881458548bc621ea2#diff-5621bd7804f10985a5f14a01ee66b14439494aa0a6cdfa6c6debd151e7c90d7aR10

Likewise for `compile-windows.sh`.

Reply:
Correct. This mistake was in part what motivated the addition of the `check_var_opt`/`check_var_req` functions in the referenced commit.